#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("bin_size", int_t, 0, "the number of bins", 10, 1, 255)
gen.add("pixel_min_value", int_t, 0, "", 0, 0, 255)
gen.add("pixel_max_value", int_t, 0, "", 180, 0, 255)
gen.add("debug_line_width", int_t, 0, "", 2, 1, 10)
exit(gen.generate(PACKAGE, "jsk_perception", "PolygonArrayColorHistogram"))
